web/components/ui/NotifyReminderPopup/NotifyReminderPopup.tsx

```diff
@@ -70,6 +70,7 @@ export const NotifyReminderPopup: FC<NotifyReminderPopupProps> = ({
         title={title}
         content={content}
         overlayInnerStyle={popupStyle}
+        color="rgba(120, 113, 255, 1)"
```

Comment:
Could you use the CSS color variables instead of hard coding the colors? They all start with `--theme-color-palette-xxxxx` and the text color can use `--theme-color-components-text-on-dark`.

Reply:
Oh okay. I got it to work on my local machine. I'm using `--theme-color-palette-7`, which is the closest to the purple that I used before. 

Reply:
Do I do the same for the color `white` in the `NotifyReminderPopup.module.scss` file?

Reply:
Yeah, using `--theme-color-components-text-on-dark` for the text color will allow people to change it.

Reply:
Got it